Node.js/GettingStarted/startup_prereq.sh

```diff
@@ -0,0 +1,17 @@
+cd $(dirname $0)
+sudo apt-get update && echo 'y' | sudo apt-get upgrade
```

Comment:
this suggests the sample is only for a certain set of distros (ubuntu only? debian?)

also the use of "apt-get update" might be viewed as hostile... in the cases where it does more then refresh meta lists.